package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "assertive-ts",
+  "name": "@stackbuilders/assertive-ts",
   "version": "0.0.0",
```

@JoseLion I think it would be nice if the version's value matches the release `1.0.0`, wdyt?
```suggestion
  "name": "@stackbuilders/assertive-ts",
  "version": "1.0.0",
```